README.md

```diff
@@ -149,56 +158,294 @@ This application will prompt users to fill in three or four games that they like
+   
+Set your bucket name in `config.py` within config directory.
+
+Go to the root directory and Run the following:
+
+  * Build a docker image with tag games:
+  
+  		docker build -t games .
+
+   * Run the docker container:
+   
+    docker run --env-file=config/config.env games run.py
+
+    
+Note this `run.py` depends on `src/store_data_in_s3.py`
+
+This should store your data into the S3 bucket.
+
+### 6. Initialize the database 
+
+#### Create the database with tables
+ First check step 3 and 4, make sure your:
+   * Environment variables are set.
+   * Requirements and dependencies are met.
+   * Set configurations in config.py
+ 
+To create the database in the location configured in `config.py` with tables customer,metacritic,sales and steam, run: 
+
+To create a database locally use make sure to set `local=True` in `config.py`:
+
+Note `create_db.py` depends on `src/clean_data.py` `src/helpers.py` and `src/game_database.py`
+
+Go to the root directory and run:
+
+    python create_db.py
+
+By default, `python create_db.py ` with `local=True` creates a database at `sqlite:///data/games.db` with the tables customer, metacritic, sales and steam.
 
 #### Defining your engine string 
 A SQLAlchemy database connection is defined by a string with the following format:
 
 `dialect+driver://username:password@host:port/database`
 
-The `+dialect` is optional and if not provided, a default is used. For a more detailed description of what `dialect` and `driver` are and how a connection is made, you can see the documentation [here](https://docs.sqlalchemy.org/en/13/core/engines.html). We will cover SQLAlchemy and connection strings in the SQLAlchemy lab session on 
+The `+dialect` is optional and if not provided, a default is used. For a more detailed description of what `dialect` and `driver` are and how a connection is made, you can see the documentation [here](https://docs.sqlalchemy.org/en/13/core/engines.html). 
 ##### Local SQLite database 
 
 A local SQLite database can be created for development and local testing. It does not require a username or password and replaces the host and port with the path to the database file: 
 
 ```python
-engine_string='sqlite:///data/tracks.db'
-
+engine_string='sqlite:///data/games.db'
 ```
 
-The three `///` denote that it is a relative path to where the code is being run (which is from the root of this directory).
+##### RDS Instance database:
 
-You can also define the absolute path with four `////`, for example:
+Make sure you are connected to northwestern vpn
 
-```python
-engine_string = 'sqlite://///Users/cmawer/Repos/2020-MSIA423-template-repository/data/tracks.db'
-```
+Go to step 3 and 4 and check environment variables and requirement.(check database name)
+
+ Go to the config directory and in `config.py` change local variable:
+    
+    local=False
+ 
+ Go to the root directory and create the rds instance database schema by running:
+       
+    python create_db.py
```

Comment:
this would not work for me (or the instructors) since you originally granted us only read priveleges, and not write priveleges.

Reply:
Sorry, I didn't mention it, but this is a way of reproducing your own database within your own rds instance. I will look into this.